plugin/plugins/file/service/service.go

```diff
@@ -262,12 +314,40 @@ func (fss *StreamingService) Stream(wg *sync.WaitGroup) error {
 	return nil
 }
 
-// Close satisfies the io.Closer interface, which satisfies the baseapp.StreamingService interface
-func (fss *StreamingService) Close() error {
+// Close satisfies the io.Closer interface
+func (fss *FileStreamingService) Close() error {
 	close(fss.quitChan)
 	return nil
 }
 
+// ListenSuccess returns a chan that is used to acknowledge successful receipt of messages by the external service
+// after some configurable delay, `false` is sent to this channel from the service to signify failure of receipt
+func (fss *FileStreamingService) ListenSuccess() <-chan bool {
```

Comment:
```suggestion
func (fss *FileStreamingService) FeedSuccess() <-chan bool {
```

Reply:
This function is not listening on anything

Reply:
In this case, correct, but this method is required to satisfy the expected interface